pkg/component/component.go

```diff
@@ -68,6 +68,31 @@ func CreateFromGit(client *occlient.Client, name string, ctype string, url strin
 	return nil
 }
 
+// GetComponentPorts provides map[container_name][]array_of_ports
```

Comment:
the comment says that provides `map` but this function returns a slice of strings

Reply:
Will fix it to slice